server/src/uuid/model/entity/mod.rs

```diff
@@ -484,6 +484,13 @@ impl Entity {
                 last_revision_fields.insert("cohesive".to_string(), "false".to_string());
             }
 
+            // FIXME: Do we need to fix this in the frontend?!
```

Comment:
I see this PR as a hotfix.
l actually think it is not a task of the DB Layer to like correct not supported fields.
But it is indeed easier to do it here than in the API (where it would be a more appropriate place).
Couldn't we plan also a migration for that?